crates/ruff/resources/test/fixtures/flake8_errmsg/EM.py

```diff
@@ -21,3 +21,31 @@ def f_c():
 def f_ok():
     msg = "hello"
     raise RuntimeError(msg)
+
+
+def f_unfixable():
+    msg = "hello"
+    raise RuntimeError("This is an example exception")
+
+
+def f_msg_in_nested_scope():
+    def nested():
+        msg = "hello"
+
+    raise RuntimeError("This is an example exception")
+
+
+def f_msg_in_parent_scope():
+    msg = "hello"
+
+    def nested():
+        raise RuntimeError("This is an example exception")
+
+
+def f_fix_indentation_check(foo):
+    if foo:
+        raise RuntimeError("This is an example exception")
```

Comment:
Can we ensure that we also properly handle non-indented statements? (We could even consider these unfixable, but the code _might_ panic if we're unwrapping on indentation right now.)

```
if foo: raise RuntimeError("This is an example exception")
if foo: x = 1; raise RuntimeError("This is an example exception")
```

Reply:
Yes, it does panic.

I think what we could do is to add the newline _before_ the assignment statement (currently it's _after_ the statement), but only if the indentation is not present. If the indentation is not present, then we'll use the default indentation (`checker.stylist.indentation`).

I'll test this out, otherwise we could just make it unfixable.

Reply:
It's actually unfixable -- the logic might work for the first statement, but it'll fail when there are multiple statements on the same line.